vault/resource_rabbitmq_secret_backend.go

```diff
@@ -57,28 +57,28 @@ func rabbitMQSecretBackendResource() *schema.Resource {
 			"connection_uri": {
 				Type:        schema.TypeString,
 				Required:    true,
-				ForceNew:    true,
+				ForceNew:    false,
```

Comment:
Recommend not setting the field to avoid any potential confusion, should also be in line with other non-force-new fields.
```suggestion
```